packages/eslint-plugin/rules/padding-line-between-statements/padding-line-between-statements._js_.test.ts

```diff
@@ -3,8 +3,7 @@
  * @author Toru Nagashima
  */
 
-// TODO: Stage 2: Test merged rule
-import rule from './padding-line-between-statements._js_'
```

Comment:
This rule is kinda special as the TS version does not inherit the JS at all. That cause the coverage to fail - which I think it's fine, but maybe at a comment in the JS file to remind us to remove that file in the future